### ISSUE TITLE
Abandoned project?

### ISSUE THREAD

**Issue by StrangerGithuber** [pandao/editor.md#924]
Hello
Is it active project or abandoned long time ago? I see the latest update was on May 9, 2019.
If it is abandoned then can someone suggest something similar editor which is open source and rich with functions like this?
I hope this project is not abandoned. When I tried the demo on the website then love of the first sight.
Thank you in advance all the answers.

**Comment by GroobleDierne** [comment]
Yes it seems that this project is abandoned but it's still (mostly) working.
I think you can find many alternatives by searching on github, if you want I created a fork that I maintain when I have time: https://gitlab.com/cobalt-os-team/MeditorJS
